pkg/demoinfocs/datatables.go

```diff
@@ -595,3 +596,23 @@ func (p *parser) bindGameRules() {
 		// "m_nCTTimeOuts"
 	})
 }
+
+func (p *parser) bindHostages() {
+	p.stParser.ServerClasses().FindByName("CHostage").OnEntityCreated(func(entity st.Entity) {
+		entityID := entity.ID()
+		p.gameState.hostages[entityID] = common.NewHostage(p.demoInfoProvider, entity)
+
+		entity.OnDestroy(func() {
+			delete(p.gameState.hostages, entityID)
+		})
+
+		var state common.HostageState
+		entity.Property("m_nHostageState").OnUpdate(func(val st.PropertyValue) {
+			oldState := state
+			state = common.HostageState(val.IntVal)
+			if oldState != state {
+				p.eventDispatcher.Dispatch(events.HostageStateChanged{State: state, Hostage: p.gameState.hostages[entityID]})
```

Comment:
should we also add the `oldState` as `PreviousState` to the event? might be helpful

Reply:
Added